.gitattributes

```diff
@@ -0,0 +1,3 @@
+index.html binary
```

Comment:
We don't need to. AFAIK, this doesn't change how github shows the diff, it just changes how git merge resolution works. Since it's a build artifact, we should never be modifying index.html to "fix" merge conflicts, we should be fixing index.bs and then ignoring the index.html changes.

Reply:
Got it, but could it affect git from the commandline?